packages/asc-ui/src/components/List/ListItemStyle.ts

```diff
@@ -0,0 +1,8 @@
+import styled from '@datapunt/asc-core'
+
+export type ListItemStyleProps = React.HTMLAttributes<HTMLElement>
+
+export default styled.li<ListItemStyleProps>`
+  word-break: break-word;
```

Comment:
Does this bubble down? As it will always have a `Link` or `Typography` as children

Reply:
Yes, I have tested that, it's by the way not changed :-)